src/index.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+
+class App extends Component {
+  render() {
+    return (
+      <div>Hello World! How's it going?</div>
+    );
+  }
+}
+
```

Comment:
I think github is getting on you for not 'wrapping' your div contents in divs.

I mean you *are* but you aren't.

You need the opening and closing div tags one line and one live bellow the text: 'Hello World!'